miden/src/main.rs

```diff
@@ -49,7 +48,7 @@ impl Cli {
 /// Executable entry point
 pub fn main() {
     // read command-line args
-    let cli = Cli::from_args();
+    let cli = Cli::try_parse().unwrap();
```

Comment:
Should this be `let cli = Cli::parse();`? This way, we'd get a pretty-printed message if arguments were not supplied correctly.